soh/src/code/z_message_PAL.c

```diff
@@ -951,7 +957,11 @@ void Message_DrawText(GlobalContext* globalCtx, Gfx** gfxP) {
                         }
                     }
                     i = j - 1;
-                    msgCtx->textDrawPos = i + 1;
+                    if (CVar_GetS32("gFastText", 0) !=0)
+                        textSpeed = fastText;
+                    else
+                        textSpeed = slowText;
+                    msgCtx->textDrawPos = i + textSpeed;
```

Comment:
If you follow my recommendations in https://discord.com/channels/808039310850130000/936124566764392510/964589497025200238, CVar_GetS32 will return the actual value to add to i, so all you'd need to change would be 

msgCtx->textDrawPos = i + 1;

into 

msgCtx->textDrawPos = i + CVar_GetS32("gTextSpeed", 1);

assuming you also renamed fast_text into text_speed (and thus gFastText into gTextSpeed).